src/gpgpu-sim/gpu-sim.h

```diff
@@ -540,7 +540,7 @@ class gpgpu_sim : public gpgpu_t {
            (m_config.gpu_max_completed_cta_opt &&
             (gpu_completed_cta >= m_config.gpu_max_completed_cta_opt));
   }
-  void print_stats();
+  void print_stats(unsigned long long streamID = -1);
```

Comment:
Same, lets remove the default value and require a streamID always

Reply:
gpgpu_sim::print_stats() is called with no parameters in gpgpusim_entrypoint.cc, stream_manager.cc, and power_stats.cc as well. If we remove this default value, do you want to pass streamID to all invocations in these three files from Accel-Sim?

Reply:
Yes. Everytime this function is called when a kernel is finished, so there should always be a stream id available. 

It can be tricky for the one inside gpgpusim_entrypoint because I think the kernel info is deleted before print. So you cannot get the stream id from kernel info anymore. If that is the case, just let me know, and add an `assert(0)` there so I know I have to fix it. It should only affect PTX mode. So you should be fine testing stuff with trace mode.

Reply:
Addressed in commit e8cc02d, assert(0) added to gpgpusim_entrypoint.cc and power_stats.cc. Please fix after merge.